db/dcrsqlite/sqlite.go

```diff
@@ -703,13 +690,13 @@ func (db *DB) RetrieveBlockSizeRange(ind0, ind1 int64) ([]int32, error) {
 	}
 	if N < 0 {
 		return nil, fmt.Errorf("Cannot retrieve block size range (%d<%d)",
-			ind1, ind0)
+			ind0, ind1)
```

Comment:
This one was saying why, so the message here was right.

Reply:
How about to change it to the `"Cannot retrieve block size range [%d,%d]" `?

Reply:
Sure, that's fine.